app/controllers/users/sp_handoff_bounced_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Users
   class SpHandoffBouncedController < Devise::SessionsController
     def bounced
-      analytics.track_event(Analytics::SP_HANDOFF_BOUNCED_VISIT)
+      analytics.sp_handoff_bounced_visit()
```

Comment:
ditto, no empty parens
```suggestion
      analytics.sp_handoff_bounced_visit
```